File: 백준/Bronze/34824. 연대 다음 고대/연대 다음 고대.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int n;
    bool iskor=false;
    cin>>n;
    vector<string> arr(n);
    for(int i=0;i<n;i++) {
        cin>>arr[i];
        if(arr[i]=="korea")
            iskor=true;
        if(arr[i]=="yonsei") {
            cout<<(!iskor?"Yonsei Won!":"Yonsei Lost...");
            break;
        }
    }
    return 0;
}

File: 백준/Bronze/34824. 연대 다음 고대/연대 다음 고대.py
import sys
input=lambda:sys.stdin.readline().rstrip()
n,iskor=int(input()),False
arr=[input() for i in range(n)]
for c in arr:
    if c=="korea":
        iskor=True
    if c=="yonsei":
        print("Yonsei Won!" if iskor==False else "Yonsei Lost...")
        break